dart/lib/src/utils/stack_trace_utils.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:meta/meta.dart';
+
+@internal
+extension StackTraceUtils on String {
```

Comment:
Let's create a static method or util class instead, related to https://github.com/getsentry/sentry-dart/issues/1471